pandas/_libs/lib.pyx

```diff
@@ -2307,6 +2305,24 @@ def tuples_to_object_array(ndarray[object] tuples):
 
 
 def to_object_array_tuples(rows: list):
+    """
+    Convert a list of tuples into an object array. Any subclass of
+    tuple in `rows` will be casted to tuple.
```

Comment:
looks like you copied the doc-string from above, can you correct it to this function

Reply:
The doc-string is copied from above, and I have correct it for `tuple`. Is it ok to see "convert a list of tuples into an object array"? Since we say "convert a list of lists ..." above.

Reply:
the parameters are wrong

Reply:
Seems the diff that you were commenting on is out-of-date?

Reply:
I see rows and a min_width parameter here, but to_object_array_tuples *only* takes rows.

Reply:
Got it, sorry for the careless. The `min_width` and is only needed for `to_object_array` and I have removed the extra docstring in `to_object_array_tuples`.